docs/contributing/pages/variables.mdx

```diff
@@ -17,7 +18,10 @@ JavaScript SDK: {{\@inject packages.version("sentry.javascript.browser") }}
 
 In this case, we expose `packages` as an accessor in the package registry which is why there is a `packages.version` function available. Additional, we expose a default context variable of `page` which contains the frontmatter of the given markdown node. For example, `{{\@inject page.title }}`.
 
-When a variable call is invalid (or errors), or doesn't match something in the known scope, it will produce an error. If you need to escape an expression you can add a leading `\` to the `@inject`.
+When a variable call is invalid (or errors), or doesn't match something in the known scope, it will produce an error message in the output in development mode,
+and force the build to fail in production.
+
+If you need to escape an expression you can add a leading `\` to the `@inject`.
```

Comment:
```suggestion
When a variable call is invalid (or errors), or doesn't match something in the known scope, it will produce an error message in the output in development mode. It will force the build to fail in production.

To escape an expression add a leading `\` to the `@inject`.
```